### ISSUE TITLE
Check failed: dilated_ksize_y <= AddPad(dshape[2], param_.pad[0]) (5 vs. 3) kernel size exceed input

### ISSUE THREAD

**Issue by 546492928** [apache/incubator-mxnet#6913]
R-3.4.0 on windows 8.1 x64  mxnet 0.10.1
When I run codes as follows, I met problems.

train <- data.matrix(w2ctrainset)
test <- data.matrix(w2ctestset)
train.x <- train[,-1]
train.y <- train[,1]
#train.x <- t(train.x/10)
train.x <- t(train.x)
train.array <- train.x
dim(train.array) <- c(10,10,1,ncol(train.x))
data <- mx.symbol.Variable("data")
#first cnn#
conv1 <- mx.symbol.Convolution(data=data,kernel=c(5,5),num_filter=20)
tanh1 <- mx.symbol.Activation(data=conv1,act_type="tanh")
pool1 <- mx.symbol.Pooling(data=tanh1,pool_type="max",kernel=c(2,2),stride=c(2,2))
#second cnn#
conv2 <- mx.symbol.Convolution(data=pool1,kernel=c(5,5),num_filter=50)
tanh2 <- mx.symbol.Activation(data=conv2,act_type="tanh")
pool2 <- mx.symbol.Pooling(data=tanh2,pool_type="max",kernel=c(2,2),stride=c(2,2))
#first fullc#
flatten <- mx.symbol.Flatten(data=pool2)
fc1 <- mx.symbol.FullyConnected(data=flatten,num_hidden=500)
tanh3 <- mx.symbol.Activation(data=fc1,act_type="tanh")
#second fullc#
fc2 <- mx.symbol.FullyConnected(data=tanh3,num_hidden=10)
#loss#
lenet <- mx.symbol.SoftmaxOutput(data=fc2)  
#train#
mx.set.seed(0)
model <- mx.model.FeedForward.create(lenet, X=train.array, y=train.y, ctx=mx.cpu(), num.round=20, array.batch.size=100, learning.rate=0.05, momentum=0.9, eval.metric=mx.metric.accuracy)

The error messages are as follows,
> model <- mx.model.FeedForward.create(lenet, X=train.array, y=train.y, ctx=mx.cpu(), num.round=20, array.batch.size=100, learning.rate=0.05, momentum=0.9, eval.metric=mx.metric.accuracy)
[13:58:17] D:\Program Files (x86)\Jenkins\workspace\mxnet\mxnet\dmlc-core\include\dmlc/logging.h:304: [13:58:17] d:\program files (x86)\jenkins\workspace\mxnet\mxnet\src\operator\./convolution-inl.h:442: Check failed: dilated_ksize_y <= AddPad(dshape[2], param_.pad[0]) (5 vs. 3) kernel size exceed input
Error in symbol$infer.shape(list(...)) : 
  Error in operator convolution1: [13:58:17] d:\program files (x86)\jenkins\workspace\mxnet\mxnet\src\operator\./convolution-inl.h:442: Check failed: dilated_ksize_y <= AddPad(dshape[2], param_.pad[0]) (5 vs. 3) kernel size exceed input
> 

I don't where I was wrong. The original "w2ctrainset"  has 187038 obs of 101variables. All in it are numbers. 
Please help me!!!

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
